_data/wai-evaluation-tools-list/submissions/4a613220-cf28-11ef-91aa-ddc548a5022b.json

```diff
@@ -0,0 +1,52 @@
+{
+  "a11yloc": "https://pac.pdf-accessibility.org/_Resources/Persistent/a/c/a/3/aca34de96ee6ab9c6273040e8c883e822a4bd602/PAC%20Accessibility%20Conformance%20Report.pdf",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results"
+  ],
+  "automated": [
+    "Entire website / software / product"
+  ],
+  "comments": "Supported languages:\r\nEnglish\r\nGerman\r\nFrench\r\nSpanish\r\nDutch",
```

Comment:
@josehilera Is there a reason why the vendor did not list these languages in the `language` field?